utils/timeutils/time.go

```diff
@@ -46,3 +47,19 @@ func (t *Time) UnmarshalYAML(unmarshal func(interface{}) error) error {
 
 	return t.UnmarshalJSON([]byte(b))
 }
+
+func GetTickAtTime(timestamp string) int64 {
```

Comment:
We shouldn't put a non-generalized function in our utils, this is very specific to the stock usecase. We could maybe include it in the api or game package. Also, we should definitely handle/return errors when things go wrong.

Reply:
Okay yeah you're right that makes a lot of sense 😳